### PR TITLE
Done the Review fixes of a PR #284

### PR DESCRIPTION
**Pull request by Nityanand13**
[PR #284](https://github.com/mattermost/mattermost-plugin-zoom/pull/284)

### CODE REVIEW

File: server/command.go [hunk 1]
```diff
@@ -153,26 +153,26 @@ func (p *Plugin) runStartCommand(args *model.CommandArgs, user *model.User, topi
...
+	userPMISettingPref, err := p.getPMISettingData(user.Id)
+	if err != nil {
+		p.askPreferenceForThisMeeting(user.Id, args.ChannelId)
 		return "", nil
 	}
 
 	switch userPMISettingPref {
 	case zoomPMISettingValueAsk:
-		p.askUserForMeetingPreference(user.Id, args.ChannelId)
+		p.askPreferenceForThisMeeting(user.Id, args.ChannelId)
```
Comment (Kshitij-Katiyar) [inline server/command.go]: Is this name given by them if not 
```suggestion
		p.askPreferenceForMeeting(user.Id, args.ChannelId)
```

File: server/command.go [hunk 2]
```diff
@@ -244,7 +244,7 @@ func (p *Plugin) runDisconnectCommand(user *model.User) (string, error) {
 
 // runHelpCommand runs command to display help text.
 func (p *Plugin) runHelpCommand() (string, error) {
-	text := starterText + strings.ReplaceAll(helpText+settingHelpText, "|", "`")
+	text := starterText + strings.ReplaceAll(helpText+"\n"+settingHelpText, "|", "`")
```
Comment (Kshitij-Katiyar) [inline server/command.go:247]:
```suggestion
	text := starterText + strings.ReplaceAll(helpText + "\n" + settingHelpText, "|", "`")
```

File: server/http.go [hunk 3]
```diff
@@ -161,8 +161,8 @@ func (p *Plugin) setPMI(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	action := postActionIntegrationRequest.Context[Action].(string)
-	mattermostUserID := r.Header.Get(MattermostUserID)
+	action := postActionIntegrationRequest.Context[actionForContext].(string)
+	mattermostUserID := r.Header.Get(mattermostUserIDHeader)
```
Comment (Kshitij-Katiyar) [inline server/http.go]: Keep this constant as it is because it will be used outside this file as well.

File: server/manifest.go [hunk 4]
```diff
@@ -7,5 +7,5 @@ var manifest = struct {
 	Version string
 }{
 	ID:      "zoom",
-	Version: "1.5.2-beta",
```
Comment (Kshitij-Katiyar) [inline server/manifest.go:10]: have you generated this file or you have edited this because you should edit manifest file.

Comment (Nityanand13) [reply server/manifest.go:10]: I haven't edited this file. Actually I have changed the version in `plugin.json` and run a command `make dist` then this file gets generated automatically.

**Review by Kshitij-Katiyar — APPROVED** [review]

File: server/command.go [hunk 5]
```diff
@@ -153,26 +153,26 @@ func (p *Plugin) runStartCommand(args *model.CommandArgs, user *model.User, topi
...
+		p.askPreferenceForMeeting(user.Id, args.ChannelId)
 		return "", nil
 	case "", trueString:
 		meetingID = zoomUser.Pmi
 	default:
-		meetingID, createMeetingErr = p.createMeetingWithoutPMI(user, zoomUser, args.ChannelId, DefaultMeetingTopic)
+		meetingID, createMeetingErr = p.createMeetingWithoutPMI(user, zoomUser, args.ChannelId, defaultMeetingTopic)
 		if createMeetingErr != nil {
-			return "", errors.New("error while creating a new meeting")
+			return "", errors.New("failed to create the meeting")
```
Comment (manojmalik20) [inline server/command.go]:
```suggestion
			return "", errors.Wrap(createMeetingErr, "failed to create the meeting")
```

**Review by manojmalik20 — COMMENTED** [review]
All the review comments are not covered in this PR. Also, why are there 42 commits?

**Review by manojmalik20 — APPROVED** [review]
Please reply to all the questions on the parent PR.

**Pull request merged**